src/components/Home.jsx

```diff
@@ -1,7 +1,44 @@
-import { Link } from "react-router-dom";
-
 const Home = ({ articles }) => {
-  return <main id="homeMain">test</main>;
+  return (
```

Comment:
I would add loading state in here too to let the user know when the page is loading as opposed to not working